### DISCUSSION TITLE
Is it fine to publish 2 versions of my extension, one for internal testing and the other for public use?

### DISCUSSION THREAD

**Discussion by chrisshaddad** [microsoft/MicrosoftEdge-Extensions#89] [Q&A]
Would that break any policies related to edge extensions? As I couldn't find anything related to that in the docs other than to sideload extensions for testing.

The plan is to have one extension hidden from the addon store which would be used for internal testing, and the other published to the store for public use.

For example for [chrome](https://developer.chrome.com/docs/webstore/cws-dashboard-distribution/#publishing-a-test-version) they mention that it's possible, and how to indicate that one version is for testing.

**Comment by sivsouvamMSFT** [comment]
Hi @chrisshaddad 

Thank you for reaching out to us. Currently, we don't have such a feature but you can utilise the listing feature Public and Private to test your extension.

**Reply by chrisshaddad** [reply]
Thx for the response @sivsouvamMSFT!

I understand it's not a built in feature, but would it be possible to create 2 separate extensions, one for testing and the other for public use? So the one used for testing will be a different extension.

Or would it mean if I create another semi identical extension used for testing (which would be private and might be versions ahead of the prod extension), it would lead to the testing extension not getting approved? Or possibly worse having both the extensions removed because they are too similar?

We need this testing version as it's hooked to an entirely different environment.
